frontend/src/scenes/project/Settings/index.js

```diff
@@ -97,26 +95,19 @@ function _Setup() {
                 <h2 className="subtitle">PostHog Toolbar</h2>
                 <ToolbarSettings />
                 <Divider />
-                {(!user.is_multi_tenancy || featureFlags['session-recording-player']) && (
-                    <>
-                        <h2 id="sessionrecording" className="subtitle">
-                            Session recording <span style={{ fontSize: 16, color: 'var(--warning)' }}>BETA</span>
-                        </h2>
-                        <p>
-                            Watch sessions replays to see how users interact with your app and find out what can be
-                            improved.
-                        </p>
-                        <OptInSessionRecording />
-                        <br />
-                        <p>
-                            This is a new feature of PostHog. Please{' '}
-                            <a href="https://github.com/PostHog/posthog/issues/new/choose" target="_blank">
-                                share feedback
-                            </a>{' '}
-                            with us!
-                        </p>
-                    </>
-                )}
+                <h2 id="sessionrecording" className="subtitle">
+                    Session recording <span style={{ fontSize: 16, color: 'var(--warning)' }}>BETA</span>
+                </h2>
+                <p>Watch sessions replays to see how users interact with your app and find out what can be improved.</p>
+                <OptInSessionRecording />
```

Comment:
Unrelated to this PR but it'd be great if we show a toast when the setting is saved (i.e. there's no feedback right now if the setting was properly updated)